staker/block_validator.go

```diff
@@ -104,9 +104,6 @@ type BlockValidator struct {
 	// For troubleshooting failed validations
 	validationInputsWriter *inputs.Writer
 
-	// For troubleshooting failed validations
-	validationInputsWriter *inputs.Writer
```

Comment:
nit: why did we delete this?

Reply:
This is an upstream update. I've checked this file is same as celestia

Reply:
@ImJeremyHe  I see it here: https://github.com/celestiaorg/nitro/blob/v3.6.7/staker/block_validator.go#L105 in. the upstream code

Reply:
I see. You can just check the code of previous lines. There is already a `validationInputsWriter`. This file is same with upstream, but when it attempts to merge to `celestia-integration`, some strange diff occurs. I am not sure how git generates the diff.  Really confusing.

I think a better way to review this PR is by comparing with upstream. Here is the link:

https://github.com/celestiaorg/nitro/compare/v3.6.7...EspressoSystems:nitro-espresso-integration:celestia-v3.6.7-merge